tests/test_outputs/test_outputs_helpers.py

```diff
@@ -0,0 +1,38 @@
+from msilib import schema
```

Comment:
I think this line can be taken out - the tests run fine without it, and I think it's the line causing the checks to fail in the PR

Reply:
Yes, thanks - that worked!